src/config/MedlemskapConfig.tsx

```diff
@@ -0,0 +1,31 @@
+import { IKategori, ISpørsmål } from '../models/spørsmal';
+
+export const SpørsmålOgSvar: ISpørsmål[] = [
+  {
+    spørsmål_id: 'søkerOppholderSegINorge',
+    tekstid: 'medlemskap.spm.opphold',
+    kategori: IKategori.Medlemskap,
+    svaralternativer: [
+      { value: 'ja', tekstid: 'svar.ja' },
+      { value: 'nei', tekstid: 'svar.nei' },
+    ],
+  },
+  {
+    spørsmål_id: 'søkerBosattINorgeSisteTreÅr',
+    tekstid: 'medlemskap.spm.bosatt',
+    kategori: IKategori.Medlemskap,
+    svaralternativer: [
+      { value: 'ja', tekstid: 'svar.ja' },
+      { value: 'nei', tekstid: 'svar.nei' },
+    ],
+  },
+  {
+    spørsmål_id: 'søkerErFlyktning',
+    tekstid: 'medlemskap.spm.flyktning',
+    kategori: IKategori.Medlemskap,
+    svaralternativer: [
+      { value: 'ja', tekstid: 'svar.ja' },
+      { value: 'nei', tekstid: 'svar.nei' },
+    ],
+  },
+];
```

Comment:
Har lagt spørsmålsconfig i en egen fil for nå. Denne kommer til å bli veldig lang senere. Er det bedre å ha alle spørsmålsobjekter i en fil  og sortere på "kategori" (f.eks. medlemskap) eller ha spørsmål allerede sortert i hver sine egne filer. ?

Reply:
For nå så har jeg bare lagt til Kategori for å skille på spørsmålene per komponent. 
Trooor ikke at vi vil ha behov for å ha en konstant med alle spørsmålene samlet for så å sortere gjennom lista hver gang for å få ut riktig kategori av spørsmål. Så det kan være mer oversiktlig å ha dem i hver sin fil. Men tenkte jeg skulle ta det opp her med dere først. Tenker dere at vi bare skal splitte det i filer fra starten av?